### ISSUE TITLE
Silex end of life

### ISSUE THREAD

**Issue by Mariyo** [bolt/bolt#7292]
Hello,

do you have any plans when Silex will end in [June 2018](http://symfony.com/blog/the-end-of-silex?utm_source=feedburner&utm_medium=twitter&utm_campaign=Feed%3A+symfony%2Fblog+%28Symfony+Blog%29)?

Thx

**Comment by GawainLynch** [comment]
Yes, see the public announcement https://twitter.com/BoltCM/status/931094453707792386

**Comment by jjmpsp** [comment]
I just discovered Bolt and I'm impressed with the codebase and the community that has grown around the framework. I came here to search for this exact issue and I'm glad to see you guys have addressed this early. 

I'm not too sure of the desired implementation, but have you considered using [Slim microframework](https://www.slimframework.com/)? It seems like a good direction to go if you want to keep the whole "fast and lightweight" approach. Slim is also fully PSR compliant and usually has 100% test coverage.
